components/formatter/RancherClusterName.vue

```diff
@@ -0,0 +1,53 @@
+<script>
+import LinkDetail from '@/components/formatter/LinkDetail';
+
+export default {
+  components: { LinkDetail },
+  props:      {
+    value: {
+      type:     String,
+      default: ''
+    },
+    row: {
+      type:     Object,
+      required: true
+    },
+    reference: {
+      type:    String,
+      default: null,
+    }
+  },
+  computed: {
+    inactiveNodes() {
+      return this.row.inactiveNodes.map(n => n.nameDisplay);
+    }
+  }
+};
+</script>
+
+<template>
+  <span class="rancher-cluster-name">
+    <v-popover v-if="inactiveNodes.length > 0" placement="top" trigger="hover" popover-class="tooltip-popover">
+      <i class="icon icon-alert text-error" />
```

Comment:
Would the bell icon be better on the right of the cluster name? This would match ember and ensure all names are aligned

Reply:
That was an experiment that was discussed with Lauren. We settled on 
![Screenshot from 2021-08-06 13-58-40](https://user-images.githubusercontent.com/55104481/128570399-cf827270-a00b-4384-adab-4b8b89c4f0d5.png)